mozdownload/cli.py

```diff
@@ -119,6 +119,10 @@ def parse_arguments(argv):
                         help='Version of the application to be downloaded for release '
                              'and candidate builds (special values: %s)' % ', '.join(
                                 scraper.RELEASE_AND_CANDIDATE_LATEST_VERSIONS.keys()))
+    parser.add_argument('--print-url',
```

Comment:
nit: All the arguments are sorted alphabetically. Can you please move this block up? Thanks.